doc/oper.xml

```diff
@@ -1503,3 +1503,47 @@ true
   </Description>
 </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="AsSemigroup">
+  <ManSection>
+    <Func Name="AsSemigroup" Arg="filt, digraph"/>
+    <Func Name="AsMonoid" Arg="filt, digraph"/>    
+    <Returns> A semilattice of partial perms. </Returns>
+    <Description>
+      The function <C>AsSemigroup</C> requires that filt be equal to
+      <Ref Func="IsParitalPermSemigroup" BookName="ref"/>.
+      If <A>digraph</A> is a
+      <Ref Prop="IsJoinSemilatticeDigraph"/> or <Ref Prop="IsLatticeDigraph"/>
+      then <C>AsSemigroup</C> returns a semigroup of partial perms which is
+      isomorphic to the semigroup whose elements are the vertices of
+      <A>digraph</A> with the binary operation
+      <Ref Oper="PartialOrderDigraphJoinOfVertices"/>. If <A>digraph</A>
+      satisfies  <Ref Prop="IsMeetSemilatticeDigraph"/> but not
+      <Ref Prop="IsJoinSemilatticeDigraph"/> then
+      <C>AsSemigroup</C> returns a semigroup of partial perms which is
+      isomorphic to the semigroup whose elements are the vertices of
+      <A>digraph</A> with the binary operation
+      <Ref Oper="PartialOrderDigraphMeetOfVertices"/>. <P/>
```

Comment:
You need to add Label="for a digraph and two vertices" (without line breaks) to this, ie
```
<Ref Oper="PartialOrderDigraphMeetOfVertices"
        Label="for a digraph and two vertices" />
```